docs/Contribute-to-documentation.md

```diff
@@ -0,0 +1,16 @@
+
+---
+## Contribute to documentation
```

Comment:
The filename of docs should be all lower case.

Reply:
fixed